bibliopixel/__init__.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
No need to add this blank line, particularly since none of the other new files starts with a blank line!